configure.ac

```diff
@@ -70,6 +70,7 @@ AC_ARG_ENABLE([direct],
 
 dnl Checks for programs
 AC_PROG_CC_C99
+AM_PROG_CC_C_O
```

Comment:
@pmmccorm I've tested with various am versions and I don't think it's an issue right now. We can't easily remove it, but the versions of automake > 1.14 that we have are not generating warnings/errors. We'll handle this at some later time when it's more pressing.